src/main/java/duke/MainWindow.java

```diff
@@ -0,0 +1,54 @@
+package duke;
```

Comment:
You could package GUI separately with the remaining classes to achieve easier management.